internal/services/provisionner.go

```diff
@@ -481,15 +481,26 @@ func updateExistingNamespace(project *v12.Project, api v13.CoreV1Interface) erro
 func generateNamespaceLabels(project *v12.Project) (labels map[string]string) {
 
 	defaultLabels := map[string]string{
-		"name":        project.Name,
-		"type":        "customer",
-		"creator":     "kubi",
-		"environment": project.Spec.Environment,
+		"name":                               project.Name,
+		"type":                               "customer",
+		"creator":                            "kubi",
+		"environment":                        project.Spec.Environment,
+		"pod-security.kubernetes.io/enforce": GetPodSecurityStandardName(project.Name),
+		"pod-security.kubernetes.io/warn":    string(utils.Config.PodSecurityAdmissionWarning),
+		"pod-security.kubernetes.io/audit":   string(utils.Config.PodSecurityAdmissionAudit),
 	}
 
 	return utils.Union(defaultLabels, utils.Config.CustomLabels)
```

Comment:
I am not very familiar with the code base, so I might need to check what this does. That's why I said someone else should also review. AT FIRST SIGHT, it looks lik ethis would work.